.golangci.json

```diff
@@ -3,7 +3,6 @@
 		"disable-all": true,
 		"enable": [
 			"govet",
-			"golint",
```

Comment:
Why remove this

Reply:
this repo is archived and using govet is sufficient enough. https://github.com/golang/lint